chart/flux/templates/secret.yaml

```diff
@@ -3,5 +3,8 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ template "flux.fullname" . }}-git-deploy
+  {{- if .Values.secret.annotations }}
```

Comment:
I am on the fence if the `.secret` provides sufficient context about what secret the annotation is added to. Looking at the way things are composed based on the configured and/or default values, I am wondering if i.e. `.ssh.secret.annotations` would make more sense (as `.git.secret` would conflict with the git-secrets feature).

Reply:
I don't have any strong opinion, and I'm happy to go with `ssh.secret.annotations`.

Reply:
:+1:

Reply:
Done. Please tell me if you want me to improve the comment or description.